modules/AOW_Actions/AOW_Action.php

```diff
@@ -107,7 +107,18 @@ public function save_lines($post_data, $parent, $key = '')
                                 if ($post_data[$key . 'param'][$i]['value_type'][$p_id] == 'Value' && is_array($p_value)) {
                                     $param_value[$p_id] = encodeMultienumValue($p_value);
                                 }elseif($post_data[$key . 'param'][$i]['value_type'][$p_id] == 'Value'){
-                                    $param_value[$p_id] = fixUpFormatting($params["record_type"], $post_data[$key . 'param'][$i]["field"][$p_id], $p_value);
+                                    if (isset($params['rel_type']) && !empty($params['rel_type']) && ($params['rel_type'] != $params['record_type'])) {
```

Comment:
Hi @SinergiaCRM, thank you for your PR.

Could we use `!==` instead of `!=` in `$params['rel_type'] != $params['record_type'])`?

Reply:
Done!

Reply:
Thank  you @SinergiaCRM